ubuntudesign/web-style-guide
# Convert tabs to 2 spaces in scss files

Spacing in scss files looks like a mess in commits, because there are tabs littered everywhere.

This replaces all tab characters with two spaces in SCSS files.

Done
---

- Replaced all tab characters with 2 spaces in scss files

QA
---

- Run `gulp sass`, make sure it still compiles
- Ensure you're okay with the concept of having to merge all the files with everyone else's work
